lib/pocolog/file_index_builder.rb

```diff
@@ -27,21 +27,27 @@ def self.file_index_builder(block_stream)
         # therefore only stores the declaration block and index map
         raw_stream_info = []
         block_pos = block_stream.tell
+        interval_rt = {}
         while (block = block_stream.read_next_block_header)
             stream_index = block.stream_index
+            interval_rt[stream_index] ||= {}
```

Comment:
why isnt it an array?

Reply:
Ultimately does not really matter, but it is less confusing with an array. Updated.